patches/tModLoader/Terraria/UI/ItemSlot.cs.patch

```diff
@@ -101,6 +101,18 @@
  				return false;
  
  			item.favorited = !item.favorited;
+@@ -443,8 +_,9 @@
+ 
+ 		if (Main.cursorOverride == 7) {
+ 			if (context == 29) {
+-				Item item2 = new Item();
++				Item /*item2 = new Item();
+-				item2.SetDefaults(inv[slot].netID);
++				item2.SetDefaults(inv[slot].netID);*/
```

Comment:
Block comments on new lines to reduce patch size (ideally no 'removals') makes it easier to maintain